WooCommerce/Classes/Authentication/Jetpack Setup/Native Jetpack Setup/JetpackSetupViewModel.swift

```diff
@@ -264,7 +264,13 @@ private extension JetpackSetupViewModel {
         currentSetupStep = .connection
         connectionType = .web
         trackSetup()
-        let action = JetpackConnectionAction.fetchJetpackConnectionURL { [weak self] result in
+        let usingApplicationPassword: Bool = {
+            if case .some(.applicationPassword) = stores.sessionManager.defaultCredentials {
+                return true
+            }
+            return false
+        }()
```

Comment:
This condition doesn't work as expected when the user is signed in using site credentials natively, as we'll get `.wporg` credentials in this case:

<img width="520" alt="Image" src="https://github.com/user-attachments/assets/881affd2-57a5-40a3-8433-1ae47c54d736" />

The same issue also occurs in tracking [here](https://github.com/woocommerce/woocommerce-ios/blob/deb7e6bec623c37705bd76272b272e40b54aabcb/WooCommerce/Classes/Analytics/WooAnalyticsEvent%2BJetpackSetup.swift#L70-L78) and I missed during the review of the previous PR (sorry).